### ISSUE TITLE
Ruby version is hard to get working

### ISSUE THREAD

**Issue by jgraham** [mystor/phlay#17]
So to get the ruby version working, I had to

```
sudo apt install ruby ruby-dev
gem install bundler --user-install
export GEM_HOME=/home/jgraham/.gem/ruby/2.5.0
```

I'm not really sure why setting `GEM_HOME` is necessary, but without it I got errors about trying to write to the system gem directory.

**Comment by mystor** [comment]
@jgraham I've pushed a new version in #16 which stops using bundler/inline, instead simply using a bash script. I'd be curious how well it works for you without GEM_HOME.

**Comment by mystor** [comment]
Fixed by #13
